components/public-api/gitpod/v1/user.proto

```diff
@@ -0,0 +1,206 @@
+syntax = "proto3";
+
+package gitpod.v1;
+
+import "gitpod/v1/editor.proto";
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/gitpod-io/gitpod/components/public-api/go/v1";
+
+service UserService {
+  // GetAuthenticatedUser allows to retrieve the current user.
+  rpc GetAuthenticatedUser(GetAuthenticatedUserRequest) returns (GetAuthenticatedUserResponse) {}
+}
+
+message GetAuthenticatedUserRequest {}
+message GetAuthenticatedUserResponse {
+  User user = 1;
+}
+
+message User {
+  // id is a UUID of the user
+  string id = 1;
+
+  // organization_id is the id of the organization this account is owned by.
+  //
+  // +optional if not set, this account is owned by the installation.
+  string organization_id = 2;
+
+  // name is the username
+  string name = 3;
+
+  // avatar_url is a link to the user avatar
+  string avatar_url = 4;
+
+  // created_at is the creation time
+  google.protobuf.Timestamp created_at = 5;
+
+  repeated Identity identities = 6;
+
+  // blocked indicates whether the account is blocked, e.g. due to vialoation of ToS.
+  bool blocked = 7;
+
+  // last_verification_time is the last time this account was verified.
+  //
+  // +optional if not set, this account is not verified.
+  google.protobuf.Timestamp last_verification_time = 8;
+
+  // verification_phone_number is the phone number used for the last verification.
+  string verification_phone_number = 9;
+
+  // user global workspace timeout settings
+  //
+  // +optional
+  WorkspaceTimeoutSettings timeout_settings = 10;
+  message WorkspaceTimeoutSettings {
+    // inacitivity is the maximum time of inactivity before the workspace is stopped or paused
+    google.protobuf.Duration inactivity = 1;
+    // inacitivity is the maximum time of disconnection before the workspace is stopped or paused
+    // set to zero to disable.
+    google.protobuf.Duration disconnected = 2;
```

Comment:
We don't need this one yet.

Reply:
alternatively we drop disabled_disconnected

and disabled_disconnected is when disconnected == 0

Reply:
if it's not needed yet, we can add it, once it's needed.